bluesky/callbacks/core.py

```diff
@@ -126,8 +127,7 @@ def __init__(self, y, x=None, legend_keys=None, xlim=None, ylim=None,
         else:
             self.x = None
         self.y, *others = _get_obj_fields([y])
-        self.fig = fig
-        self.ax = fig.gca()
+        self.ax = plt.gca()
```

Comment:
I addressed this by leaving the `fig` kwarg in, with a deprecation warning, and adding `ax`. Using both `fig` and `ax` raises since that would lead to confusing behavior. I don't think we need `isinstance`; we are two separate kwargs.